zkevm-circuits/src/evm_circuit/execution.rs

```diff
@@ -1775,10 +1779,7 @@ impl<F: Field> ExecutionConfig<F> {
             .rw_indices
             .iter()
             .map(|rw_idx| block.rws[*rw_idx])
-            .map(|rw| {
-                rw.table_assignment_aux(evm_randomness)
-                    .rlc(lookup_randomness)
-            })
+            .map(|rw| rw.table_assignment_aux().unwrap().rlc(lookup_randomness))
```

Comment:
this `unwrap` is customized method in  rw.rs(https://github.com/scroll-tech/zkevm-circuits/blob/word_hi_lo/zkevm-circuits/src/witness/rw.rs#L375) instead of rust standard `unwrap`,